hub/jupyterhub_config.py

```diff
@@ -13,6 +13,28 @@
 # Our simplest user image! Optimized to just... start, and be small!
 c.KubeSpawner.singleuser_image_spec = 'yuvipanda/simple-singleuser:v1'
 
+# Configure dynamically provisioning pvc
+c.KubeSpawner.pvc_name_template = 'claim-{username}-{userid}'
+c.KubeSpawner.storage_class = 'single-user-storage'
+c.KubeSpawner.acess_modes = ['ReadWriteOnce']
+c.KubeSpawner.storage = '1GiB'
+
+# Add volumes to singleuser pods
+c.KubeSpawner.volumes = [
+	{
+		'name': 'volume-{username}-{userid}', 
+		'persistentVolumeClaim': {
+			'claimName': 'claim-{username}-{userid}'
+		}
+	}
+]
+c.KubeSpawner.volume_mounts = [
+	{
+		'mountPath': '/home', 
+		'name': 'volume-{username}-{userid}'
+	}
+]
+
```

Comment:
nit: Can you use spaces instead of tabs characters? You can use these lines in your Sublime config to automatically do that in the future:

```
"translate_tabs_to_spaces": true,
# Here are more config options that are good:
"trim_trailing_white_space_on_save": true,
"rulers":
[
	79
],
"ensure_newline_at_eof_on_save": true,
```